src/tests/suite/daos_aggregate_ec.c

```diff
@@ -785,7 +785,9 @@ test_all_ec_agg(void **statep)
 	test_half_stripe(&ctx);
 	test_partial_stripe(&ctx);
 	test_range_punch(&ctx);
-	sleep(40);
+	print_message("sleep 45 seconds for aggregation ...\n");
+	sleep(45);
+	print_message("verification after aggregation\n");
```

Comment:
probably need set pool property to "time" and use force aggregation to make sure the aggregation really happen here.

Reply:
ok, sure.